### PR TITLE
Add avatar upload 

### PR DESCRIPTION
**Pull request by nikriek**

### CODE REVIEW

File: spec/models/user_spec.rb [hunk 1]
```diff
@@ -18,7 +18,19 @@
...
 RSpec.describe User, type: :model do
   it "is valid when produced by a factory" do
-    user = FactoryBot.build(:user)
-    expect(user).to be_valid
+	user = FactoryBot.build(:user)
+	expect(user).to be_valid
+  end
+
+  it 'is valid with image as avatar' do
+
```
Comment (bakoe) [inline spec/models/user_spec.rb:26]: Don’t forget to fill this empty test with content!

File: spec/models/user_spec.rb [hunk 2]
```diff
@@ -18,7 +18,19 @@
...
+  end
+
+  it 'is not valid with any other file type than image' do
+
```
Comment (bakoe) [inline spec/models/user_spec.rb:30]: Don’t forget to fill this empty test with content!

File: spec/models/user_spec.rb [hunk 3]
```diff
@@ -18,7 +18,19 @@
...
+  end
+
+  it 'is not valid with an avatar of size >2mb' do
+
```
Comment (bakoe) [inline spec/models/user_spec.rb:34]: Don’t forget to fill this empty test with content!

**Review by bakoe — CHANGES_REQUESTED** [review]

File: spec/models/user_spec.rb (hunk 1 above)
Comment (nikriek) [reply spec/models/user_spec.rb:26]: will do :)

**Comment by nikriek** [comment]
Closing this PR to follow the branch naming schema

**Pull request closed without merging**